src/components/Select.js

```diff
@@ -245,7 +243,9 @@ class Select extends React.Component {
           fontSize: theme.FontSizes.MEDIUM,
           padding: '8px 10px',
           position: 'relative',
-          boxSizing: 'border-box'
+          boxSizing: 'border-box',
+          width: '100%',
+          textAlign: 'left'
```

Comment:
Is there any need to update border styling as well?

Reply:
Scratch this question.  I expanded the code and realized there is already border styling in place.